src/coins.ts

```diff
@@ -815,6 +815,20 @@ const basicCoins: BasicCoin[] = [
         address: '0xe71678794fff8846bff855f716b0ce9d9a78e844',
         decimals: 18,
       },
+    },
+  },
+
+  // LINEA TEST COIN
+  {
+    key: CoinKey.LNATEST,
```

Comment:
This is not a Linea specific token. It is the TEST token of Connext-Amarok. So I would include connext in the name/key instead.